tests/test_util.py

```diff
@@ -408,6 +408,16 @@ def test_get_manifest_digests_missing(tmpdir, has_content_type_header, has_conte
 
     url = 'https://example.com/v2/spam/manifests/latest'
 
+    def custom_retries(*args, **kwargs):
```

Comment:
remaining amount???
do_retry is called once and Session with Retry set to 'total=0' will be used, which mean no retry at all